Formula/fish.rb

```diff
@@ -21,16 +22,18 @@ class Fish < Formula
   end
 
   head do
-    url "https://github.com/fish-shell/fish-shell.git"
+    url "https://github.com/fish-shell/fish-shell.git", branch: "master"
 
-    depends_on "sphinx-doc" => :build
+    # The source tarballs come with prebuilt documentation.
+    # It's only needed for git checkouts, and even then it's optional,
+    # as fish will fall back to using documentation hosted in a versioned
+    # path at fish-shell.com/docs/. This can actually be safely skipped.
+    depends_on "sphinx-doc" => :reccomended
```

Comment:
```suggestion
    depends_on "sphinx-doc" => :recommended
```

Reply:
It did seem to work - the thing trying to be avoided is a dozen packages including python 3.10 just because we're compiling from git. I'll take out the dependency required if it's actually contraindicated.

Reply:
I'd rather leave this dependency alone. It's a `:build` dependency -- users can uninstall `sphinx-doc` and any dependents they weren't using afterward.

Reply:
> Note that making this :recommended will make brew prevent users from uninstalling sphinx-doc and dependencies after doing brew install --HEAD fish. It can be overridden with --force and/or --ignore-dependencies, of course, but that's not really ideal.

That is a good point and certainly not ideal. I'll adjust things.